projects/1.20/assets/gw-copper/gwcopper/lang/zh_cn.json

```
@@ -0,0 +1,35 @@
+{
+  "item.gwcopper.copper_armor_leggings": "铜护腿",
+  "gui.gwcopper.repair_stration_gui.label_repair_station": "修补站",
+  "item.gwcopper.copper_armor_helmet": "铜头盔",
+  "item.gwens_flat_survival_plus.copper_hoe": "铜锄",
+  "item.gwcopper.wrench": "扳手",
+  "item.gwens_flat_survival_plus.iron_hammer": "铁锤",
+  "item.gwens_flat_survival_plus.copper_pickaxe": "铜镐",
+  "item.gwcopper.copper_hammer": "铜锤",
+  "enchantment.gwens_flat_survival_plus.fixing_strike": "修复打击",
+  "item.gwcopper.copper_shears": "铜剪",
+  "block.gwens_flat_survival_plus.repair_station": "修补站",
+  "item.gwens_flat_survival_plus.copper_nugget": "铜锭",
+  "item.gwcopper.copper_shovel": "铜锹",
+  "item.gwens_flat_survival_plus.copper_shovel": "铜锹",
+  "subtitles.wrench_use": "使用扳手",
```

```suggestion
  "subtitles.wrench_use": "扳手：使用",
```
详见[原版字幕](https://zh.minecraft.wiki/w/%E5%AD%97%E5%B9%95)
此类问题请全文通查